charts/caddy-ingress-controller/templates/loadbalancer.yaml

```
@@ -14,7 +12,9 @@ metadata:
     {{- include "caddy-ingress-controller.labels" . | nindent 4 }}
 spec:
   type: "LoadBalancer"
+  {{- if (semverCompare "<= 1.24-0" .Capabilities.KubeVersion.Version) }}
```

the version should have the dot instead of the dash?
```suggestion
  {{- if (semverCompare "<= 1.24.0" .Capabilities.KubeVersion.Version) }}
```

---

thanks for spotting ! @mavimo